.travis.yml

```diff
@@ -30,45 +30,12 @@ script:
   - if [ -z "${SHELLCHECK-}" ]; then make TEST_SUITE=$TEST_SUITE URCHIN=/tmp/urchin/package/urchin test-$SHELL ; fi
 env:
   global:
-    - CXX=g++-4.8
-    - CC=gcc-4.8
+    - CXX=clang++-3.8
+    - CC=clang-3.8
     - PATH=$(echo $PATH | sed 's/::/:/')
     - NVM_DIR="${TRAVIS_BUILD_DIR}"
   matrix:
     - SHELLCHECK=true
     - SHELL=bash TEST_SUITE=install_script
     - SHELL=sh TEST_SUITE=fast
-    - SHELL=dash TEST_SUITE=fast
```

Comment:
You can enable your own repo on Travis and push branches to it without creating PRs, btw